usbhost.h

```diff
@@ -63,6 +63,8 @@ template< typename SPI_CLK, typename SPI_MOSI, typename SPI_MISO, typename SPI_S
                 #else
                     SPI.setClockDivider(SPI_CLOCK_DIV2); // This will set the SPI frequency to 8MHz - it could be higher, but it is not supported in the old API
                 #endif
+#elif defined(__ARDUINO_ARC__)
+                SPI.setClockDivider(SPI_CLOCK_DIV2);
```

Comment:
Is this really needed? Newer boards should support the new SPI API with transactions, where the clock speed is set like so: https://github.com/joebowbeer/USB_Host_Shield_2.0/blob/d57d810e9335bb922e199d6c32006db66372f62c/usbhost.h#L158.